crawler/crawling.py

```diff
@@ -1,14 +1,16 @@
-"""A simple web crawler -- classes implementing crawling logic."""
+"""A simple web crawler -- class implementing crawling logic."""
 
 import asyncio
 import cgi
-from http.client import BadStatusLine
+from collections import namedtuple
 import logging
 import re
 import time
 import urllib.parse
 
-logger = logging.getLogger(__name__)
+import aiohttp  # Install with "pip install aiohttp".
```

Comment:
I think it's safe to add a `requirements.txt` to the project, and to put a quick note in the readme on how to `pip install -r`.